ios/FluentUI/Core/FluentUIFramework.swift

```diff
@@ -3,6 +3,9 @@
 //  Licensed under the MIT License.
 //
 
+#if SWIFT_PACKAGE
```

Comment:
Suggestion: maybe add a comment to clarify that SWIFT_PACKAGE is defined when the code is being built by Swift Package Manager?